Source/Tools/FEXQonfig/main.qml

```diff
@@ -603,6 +603,12 @@ ApplicationWindow {
                                     tooltip: qsTr("Controls TSO emulation on memcpy/memset instructions")
                                     config: "MemcpySetTSOEnabled"
                                 }
+                                ConfigCheckBox {
+                                    leftPadding: 24
+                                    text: qsTr("Strict in-process split-lock atomics")
```

Comment:
Was this intentionally positioned in the middle of the three other options? It looks rather awkward in practice. It should probably be below the other three options.

Also, could you confirm this is only used for normal TSO operation and that it is ignored for paranoid TSO?

Reply:
Actually it is independent of the these TSO toggles. Moved it below the three radio buttons since it is still part of the memory model emulation.